apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/KeyValuePairEnumerableExtensions.cs

```diff
@@ -55,5 +57,21 @@ public static Value ToStructValue(this IEnumerable<KeyValuePair<string, object>>
             }
             return hasValues ? Value.ForStruct(fieldsStruct) : null;
         }
+
+
+        /// <summary>
+        /// Returns a string representing the given fields with the format:
+        /// ['key1'='value1']['key2'='value2']...
+        /// </summary>
+        internal static string ToDefaultStringRepresentation<TKey, TValue>(this IEnumerable<KeyValuePair<TKey, TValue>> fields)
```

Comment:
This one's okay to be an extension method if that's helpful, given that it's internal.